MBBSEmu.Tests/CPU/FST_Tests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Iced.Intel;
+using System;
+using Xunit;
+using static Iced.Intel.AssemblerRegisters;
+
+namespace MBBSEmu.Tests.CPU
+{
+    public class FST_Tests : CpuTestBase
+    {
+        [Theory]
+        [InlineData(double.MaxValue)]
+        [InlineData(double.MinValue)]
+        [InlineData(0.0d)]
+        [InlineData(double.NaN)]
+        public void FST_Test_ST1(double ST0Value)
+        {
+            Reset();
+            mbbsEmuCpuRegisters.Fpu.SetStackTop(0);
+            mbbsEmuCpuCore.FpuStack[0] = ST0Value;
+
+            var instructions = new Assembler(16);
+            instructions.fst(st0);
```

Comment:
ST0 = ST0? How about stashing in another position to verify it got copied correctly

Reply:
Good Catch -- Fixed